src/components/smartSearch/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { TIME_FRAME } from 'types/smartSearch';
+import { ZetkinSmartSearchFilter } from 'types/zetkin';
+import { ZetkinSmartSearchFilterWithId } from 'types/smartSearch';
+
+export function isFilterWithId(filter: ZetkinSmartSearchFilterWithId | ZetkinSmartSearchFilter): filter is ZetkinSmartSearchFilterWithId {
+    return (filter as ZetkinSmartSearchFilterWithId).id !== undefined;
+}
```

Comment:
I would put this in the types module, i.e. types/smartSearch.ts.

Reply:
resolved in #310